### PR TITLE
Update Make It Native release notes

### PR DESCRIPTION
**Pull request by smarki**
Updating release notes for final apps.

### CODE REVIEW

File: content/releasenotes/mobile/make-it-native-app.md [hunk 1]
```diff
@@ -10,6 +10,16 @@ tags: ["Native", "Android", "iOS", "Apple", "Android"]
 For more information on the Make It Native app, see [Getting the Make It Native App](/refguide/getting-the-make-it-native-app).
 {{% /alert %}}
 
+## Android 1.0.0 / iOS 1.0.2
+
+**Release date: August 1st, 2019**
+
+Official release of the Make It Native apps, with support for Mendix Studio Pro 8.0. The apps are not backwards compatible with the beta releases of Mendix Studio Pro.
```
Comment (Adam-Dupaski) [inline content/releasenotes/mobile/make-it-native-app.md]: -Multiple "apps" or one "app"?
-Add link to 8.0.0 RN
-"beta releases of Mendix Studio Pro" is not specific enough - what is better?

Comment (ConnorLand) [reply content/releasenotes/mobile/make-it-native-app.md]: * App point clarified
* Link added
* Language around Betas aligned to Beta language in the 8.0 release notes. Does that look better/specific enough? @Adam-Dupaski

Comment (Adam-Dupaski) [reply content/releasenotes/mobile/make-it-native-app.md]: Good. I would only change "Beta 1, 2, or 3 versions of Mendix Studio Pro" to "Mendix Studio Pro 8.0.0 beta versions" to be clear that these were 8.0.0 betas.

Comment (ConnorLand) [reply content/releasenotes/mobile/make-it-native-app.md]: Done

**Pull request merged**